tests/ci/Dockerfile-fedora

```diff
@@ -0,0 +1,9 @@
+FROM fedora
+
+RUN dnf -y update
+RUN dnf -y install dnf-plugins-core
+RUN dnf -y builddep PackageKit
+RUN dnf -y install libdnf-devel redhat-rpm-config autoconf-archive
+
```

Comment:
I'd merge lines 4 and 6 together, and also add `autoconf automake libtool` to that line.